tests/test_specifiers.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from packaging.specifiers import Specifier, SpecifierSet
+
+from passa.markers import cleanup_specs
+
+
+@pytest.mark.parametrize("spec, cleaned", [
```

Comment:
These are not valid. Which of these is a valid pyspecset from a metadata object? if the answer is 'none of them', then why are they being used as inputs to the function which is taking pyspecsets as output?